src/lib/xmlparse.rs

```diff
@@ -400,8 +400,8 @@ pub enum EncodingType {
 
 pub enum OldEncoding {
```

Comment:
Good point, I'm removing it. There might be other stuff that could be removed, but since we export all of it as public, the compiler doesn't complain.